.circleci/config.yml

```diff
@@ -1,211 +1,180 @@
 version: 2.1
 jobs:
   tarball:
-    docker:
-      - image: circleci/buildpack-deps:stretch
+    machine:
+      image: ubuntu-1604:202007-01
+#    resource_class: large
     steps:
       - checkout
       - run: sudo apt-get update && sudo apt-get install -y build-essential libfuse-dev libsqlite3-dev libgmp-dev libncurses5-dev pkg-config git g++ gcc libre2-dev python3-sphinx
-      - run: USE_FUSE_WAKE=0 make tarball && mkdir www && ./bin/wake --no-workspace --html > www/index.html
+      - run: make tarball && mkdir www && ./bin/wake --no-workspace --html > www/index.html
       - run: PATH=$(pwd)/bin:$PATH PYTHONPATH=$(pwd)/scripts python3 scripts/wake2rst.py
       - persist_to_workspace:
           root: .
           paths:
+            - .circleci/dockerfiles
             - wake_*.tar.xz
-            - wake.spec
             - debian
             - www
             - scripts/sphinx/build/html
 
   alpine:
-    docker:
-      - image: alpine:3.11.5
+    machine:
+      image: ubuntu-1604:202007-01
+#      docker_layer_caching: true
+#    resource_class: large
     steps:
-      - run: apk add g++ make pkgconf git tar xz gmp-dev re2-dev sqlite-dev fuse-dev ncurses-dev dash sqlite-static ncurses-static
       - attach_workspace:
           at: /tmp/workspace
+      - run: docker build -f /tmp/workspace/.circleci/dockerfiles/alpine -t wake-alpine .
       - run: |
-          wget https://github.com/libfuse/libfuse/releases/download/fuse-2.9.9/fuse-2.9.9.tar.gz
-          tar xvzf fuse-2.9.9.tar.gz
-          cd fuse-2.9.9
-          ./configure
-          make
-          cp lib/.libs/libfuse.a /usr/lib
-          cd ..
           tar xJf /tmp/workspace/wake_*.tar.xz
-          cd wake-*
-          USE_FUSE_WAKE=0 make static
-          cd ..
-          mkdir -p release/alpine
-          mv wake-*/wake-static_* release/alpine
+          docker run --rm --mount type=bind,source=$PWD,target=/build --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined wake-alpine make -C wake-* static
+          docker run --rm --mount type=bind,source=$PWD,target=/build wake-alpine /bin/sh -c "tar xJf wake-*/wake-static_*; ./wake-*/bin/wake --no-workspace -x 'shellJob \"ls /\" Nil | getJobStdout'"
```

Comment:
I'm going to assume that you figured out all of these flags from trial and error? Hope that wasn't too painful!

Reply:
google helped a lot. it was painful.